terraform/examples/io500/config/GCP-32C-9S16d-NOGVNIC-rf1.sh

```diff
@@ -51,13 +47,13 @@ DAOS_CLIENT_MACHINE_TYPE=c2-standard-16
 DAOS_CLIENT_GVNIC=false
 
 # Storage
-DAOS_POOL_SIZE="$(awk -v disk_count=${DAOS_SERVER_DISK_COUNT} -v server_count=${DAOS_SERVER_INSTANCE_COUNT} 'BEGIN {pool_size = 350 * disk_count * server_count / 1000; print pool_size"TB"}')"
+DAOS_POOL_SIZE="$(awk -v disk_count=${DAOS_SERVER_DISK_COUNT} -v server_count=${DAOS_SERVER_INSTANCE_COUNT} 'BEGIN {pool_size = 375 * disk_count * server_count / 1000; print pool_size"TB"}')"
```

Comment:
Does "375" work in all your tests?

Reply:
It worked when I ran `start.sh -c config/<config>` with each of the files in the `config` directory.  Just to be sure I'll go run this one again.

Reply:
Thank you!